### PR TITLE
Go doc and CRD readability improvements

### LINKED ISSUE #3924: Use real sentences for comments kube-builder "managed" objects
The comments are copied over to OpenAPI/CRD spec but that does not take in the line breaks, see https://github.com/k0sproject/k0s/pull/3902#discussion_r1448475347. We should use real sentences, with punctuation so the field comments in CRDs/OpenAPI will be more readable.

### PR DESCRIPTION
**Pull request by kke**
## Description

Adds periods to documentation comments that will be in generated documentation / CRDs and so on.

Files are also `gofumpt`ed.

Fixes #3924 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

## How Has This Been Tested?

- [ ] Manual test
- [ ] Auto test added

## Checklist:

- [ ] My code follows the style [guidelines](https://github.com/k0sproject/k0s/blob/main/docs/contributors/overview.md) of this project 
- [ ] My commit messages are [signed-off](https://github.com/k0sproject/k0s/blob/main/docs/contributors/github_workflow.md)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: cmd/airgap/listimages.go [hunk 1]
```diff
@@ -29,9 +29,9 @@ func NewAirgapListImagesCmd() *cobra.Command {
 	var all bool
 
 	cmd := &cobra.Command{
-		Use:     "list-images",
-		Short:   "List image names and version needed for air-gap install",
-		Example: `k0s airgap list-images`,
+		Use:		"list-images",
+		Short:		"List image names and version needed for air-gap install",
+		Example:	`k0s airgap list-images`,
```
Comment (kke) [inline cmd/airgap/listimages.go:34]: Hmm I don't know why it messes with these

**Comment by kke** [comment]
Hmm, should the `static` be regenerated somehow?

File: cmd/airgap/listimages.go (hunk 1 above)
Comment (kke) [reply cmd/airgap/listimages.go]: (and still don't, resorted to just `gofumpt` the file after modification)

**Comment by twz123** [comment]
The original ticket was just about making the doc strings in the OpenAPI specs easier to read. Also, there are probably some cases in which there are periods missing not only at the end, but in the middle of the comments. So more a human task than an automated one.

**Comment by kke** [comment]
Go doc comments [should](https://tip.golang.org/doc/comment) be complete sentences too. 

You could still edit the remaining problems manually after this has performed the first round of adding missing dots.

**Comment by kke** [comment]
> Upon reviewing your Kubernetes CustomResourceDefinition (CRD) YAML for clusterconfigs.k0s.k0sproject.io, there were no obvious typos or missing punctuation. - *ChatGPT*

**Pull request closed without merging**